wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/Container/Empty placeholders/EmptyConversationSearchResultsView.swift

```diff
@@ -24,40 +24,59 @@ final class EmptyConversationSearchResultsView: UIView {
 
     var newConversationAction: () -> Void
     var connectWithPeopleAction: (() -> Void)?
+    @available(*, unavailable)
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
 
     private var hostingViewController: UIHostingController<EmptyView>!
 
-    init(iPadTargeted: Bool,
-         newConversationAction: @escaping () -> Void,
-         connectWithPeopleAction: (() -> Void)?) {
+    init(
+        iPadTargeted: Bool,
+        newConversationAction: @escaping () -> Void,
+        connectWithPeopleAction: (() -> Void)?
+    ) {
         self.newConversationAction = newConversationAction
         self.connectWithPeopleAction = connectWithPeopleAction
 
         super.init(frame: .zero)
 
-        self.hostingViewController = UIHostingController(rootView: EmptyView(iPadTargeted: iPadTargeted, newConversationAction: { [weak self] in
-            self?.newConversationAction()
-        }, connectWithPeopleAction: { [weak self] in
-            self?.connectWithPeopleAction?()
-        }))
+        self.hostingViewController = UIHostingController(rootView: EmptyView(
+            iPadTargeted: iPadTargeted,
+            newConversationAction: { [weak self] in
+                self?.newConversationAction()
+            },
+            connectWithPeopleAction: { [weak self] in
+                self?.connectWithPeopleAction?()
+            }
+        ))
 
-        self.addSubview(hostingViewController.view)
+        addSubview(hostingViewController.view)
```

Comment:
Also this change, I thought the formatter had already removed this.

Reply:
Maybe it was skipped because I rebased and not merged develop into my branch.